scripts/change-log-generator-utils.js

```diff
@@ -283,6 +285,32 @@ function getReleaseDate() {
     }).format(releaseDate);
 }
 
+/**
+ *
+ * Complete the heavy lifting to update the changelog by grabbing the
+ * new commits, grouping everything, and creating the text for editing.
+ * @param {string} remoteReleaseBranch
+ * @param {string} remotePreviousBranch
```

Comment:
Comparing both of these on remote was more consistent for if I didn't have the branch locally for instance.